frontend/services/entities/hosts.ts

```diff
@@ -53,63 +68,51 @@ export default {
       },
     });
   },
-  exportHosts: (options: ILoadHostsOptions | undefined) => {
-    const { HOSTS_REPORT, LABEL_HOSTS } = endpoints;
+  exportHosts: (options: IExportHostsOptions) => {
+    const { HOSTS_REPORT } = endpoints;
+    const sortBy = options.sortBy;
     const selectedLabels = options?.selectedLabels || [];
     const globalFilter = options?.globalFilter || "";
-    const sortBy = options?.sortBy || [];
     const teamId = options?.teamId || null;
     const policyId = options?.policyId || null;
-    const policyResponse = options?.policyResponse || null;
+    const policyResponse = options?.policyResponse || "passing";
     const softwareId = options?.softwareId || null;
     const visibleColumns = options?.visibleColumns || null;
+    const orderKeyParam = `?order_key=${sortBy[0].key}`;
```

Comment:
Can you be sure that index 0 exists? I didn't see a check for length that guards for this but I might have missed it.

Reply:
I added a new interface called `IExportHostsOptions` that sets `sortBy` as a required array, so we know there will always be at least one item.

Reply:
Does TypeScript enforce length for array types? I think it is still possible to have an empty typed array.

Reply:
It doesn't automatically force a length, but I have the type of `sortBy` set to `ISortOption` which requires an object. That's what's enforcing the length.

Reply:
Testing locally just to make sure that's true :)

Reply:
And you are right. Typescript is not enforced array length. I assumed it was since it was letting me reference `sortBy[0].key` without confirming the length. Kinda seems like a TS bug IMO but great catch! 

Sorting is a required field for the report endpoint, so I'm throwing an error if sortBy is ever empty.